core/src/main/java/tc/oc/pgm/PGMAudiences.java

```diff
@@ -0,0 +1,30 @@
+package tc.oc.pgm;
+
+import net.kyori.adventure.audience.Audience;
+import net.kyori.adventure.key.Key;
+import net.kyori.adventure.platform.bukkit.BukkitAudiences;
+import net.kyori.adventure.sound.Sound;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.format.NamedTextColor;
+import org.bukkit.plugin.Plugin;
+
+public class PGMAudiences {
```

Comment:
Instead of having something like `PGMAudiences`, I would like this to live in `util`. 

`tc.oc.util.text.Audience` would be both an interface, with things like `sendWarning` wrapper, and 1-stop-shop to create audiences from players or groups of players.

Reply:
I see, any reason you decided to deprecate the provider that was in there?

The way im understanding this is that the provider also should stay in this Audience class